pkg/mtbroker/filter/filter_handler.go

```diff
@@ -187,13 +188,16 @@ func (h *Handler) ServeHTTP(writer http.ResponseWriter, request *http.Request) {
 	}
 
 	// Check if the event should be sent.
+	ctx = logging.WithLogger(ctx, h.logger)
```

Comment:
why are you adding the logger to the context?

Reply:
the filter function pulls the logger from the context

Reply:
gotcha